src/display/editor/annotation_editor_layer.js

```diff
@@ -36,6 +38,7 @@ import { setLayerDimensions } from "../display_utils.js";
  * @property {TextAccessibilityManager} [accessibilityManager]
  * @property {number} pageIndex
  * @property {IL10n} l10n
+ * @property {AnnotationLayer} annotationLayer
```

Comment:
This may not actually be available, hence:
```suggestion
 * @property {AnnotationLayer} [annotationLayer]
```